### PR TITLE
Feed cleanup

### PR DESCRIPTION
**Pull request by mmitche**
Move away from blob feeds and attempt to move away from myget feeds.

### CODE REVIEW

File: NuGet.config [hunk 1]
```diff
@@ -6,11 +6,8 @@
   <packageSources>
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
-    <add key="dotnet-core" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" />
-    <add key="dotnet-fsharp" value="https://dotnet.myget.org/F/fsharp/api/v3/index.json" />
-    <add key="dotnet-try" value="https://dotnet.myget.org/F/dotnet-try/api/v3/index.json" />
```
Comment (brettfo) [inline NuGet.config]: Both `dotnet-fsharp` and `dotnet-try` are required for this repo.

Comment (mmitche) [reply NuGet.config]: I don't think that's the case. dotnet5 has the fsharp packages and does not appear to be getting regular updates anyway.

**Comment by mmitche** [comment]
@brettfo I'm going to update System.CommandLine to a version that is released (0.3.0-alpha.20070.2). Sound good?

**Comment by brettfo** [comment]
I'll deflect the System.CommandLine question to @jonsequitur.

**Comment by mmitche** [comment]
> I'll deflect the System.CommandLine question to @jonsequitur.

FYI the current verison appears to be one day later than that.

**Comment by jonsequitur** [comment]
@mmitche System.CommandLine should be at or above version 2.0.0-beta1.20161.1. System.CommandLine.Rendering should be at or above 0.3.0-alpha.20161.1. These were published in the corefx feed, I believe.

**Comment by mmitche** [comment]
> @mmitche System.CommandLine should be at or above version 2.0.0-beta1.20161.1. System.CommandLine.Rendering should be at or above 0.3.0-alpha.20161.1. These were published in the corefx feed, I believe.

Looks like all the newer versions are on dotnet5, I'll do an update.

**Comment by brettfo** [comment]
@jonsequitur I'm OK with the changes, do the command line updates look good to you?

**Pull request merged**